### PR TITLE
fix(bi.sql.view.field): fix decorator for create

### PR DESCRIPTION
**Pull request by agb80**
Create function must be decorated with @api.model and not with
@api.multi as is now

### CODE REVIEW

**Review by astirpe — APPROVED** [review]
LGTM

File: bi_sql_editor/models/bi_sql_view_field.py [hunk 1]
```diff
@@ -108,7 +108,8 @@ def _compute_index_name(self):
                 sql_field.bi_sql_view_id.view_name, sql_field.name)
 
     # Overload Section
-    @api.multi
+    @api.model
+    @api.returns('self', lambda value:value.id)
```
Comment (legalsylvain) [inline bi_sql_editor/models/bi_sql_view_field.py:112]: i don't understand why adding the second decorator. (@api.returns). Should works without dont you think ?

other wise, LGTM.

Comment (astirpe) [reply bi_sql_editor/models/bi_sql_view_field.py:112]: @legalsylvain looking at how the create is used in the code, it seems the return value must be an id, not a recordset

Comment (agb80) [reply bi_sql_editor/models/bi_sql_view_field.py:112]: @legalsylvain just as @astirpe saids, that is how the code is decorated on main models.py odoo function

Comment (legalsylvain) [reply bi_sql_editor/models/bi_sql_view_field.py:112]: Sorry, I don't understand. The function currently return a recordset.

see : ``return super(BiSQLViewField, self).create(vals)``

So, I think that @api.model will works.

**Review by legalsylvain — COMMENTED** [review]
Could you check travis & runbot ? 

thanks.

**Pull request closed without merging**